GUI.py

```diff
@@ -369,3 +411,4 @@ def lockMultipleItems(items: list):
     
 if __name__ == "__main__":
     gui = mainGUI()
+
```

Comment:
@JackPifer Also make sure to adjust the height of the window so that the button can be seen clearly